morphic.js

```diff
@@ -9403,16 +9442,34 @@ MenuItemMorph.prototype.mouseEnter = function () {
         this.image = this.highlightImage;
         this.changed();
     }
+    if (this.doubleClickAction instanceof MenuMorph) {
+        var w=this.parentThatIsA(WorldMorph);
+        this.doubleClickAction.popup(w,
+                                     new Point(this.parent.bounds.right()-10,
+                                     w.hand.position().y-10),
+                                     true);
+        if (!contains(this.parent.children, this.doubleClickAction))
+            this.parent.addChild(this.doubleClickAction);
+    }
     if (this.hint) {
         this.bubbleHelp(this.hint);
     }
 };
 
 MenuItemMorph.prototype.mouseLeave = function () {
-    if (!this.isListItem()) {
+    if (this.doubleClickAction instanceof MenuMorph) {
+		if (!contains(this.world().hand.allMorphsAtPointer(),
```

Comment:
Yeah, as I said, where I come from tab always means eight spaces, but I set my preferences to use four spaces so there shouldn't be any new ones of these.